analyses/mutation-signatures/util/mut_sig_functions.R

```diff
@@ -0,0 +1,209 @@
+# Functions for making mutational signature plots and calculations
+#
+# C. Savonen for ALSF - CCDL
+# 2019
+
+# Load this library
+library(deconstructSigs)
+
+################################################################################
+sample_mut_sig_plot <- function(which_sig_list, label = "none", output_dir = getwd(),
+                                sample_ids) {
+  # Given a list of `deconstructSigs::WhichSignature` output, make plots for each sample using
+  # deconstructSigs::plotSignatures
+  #
+  # Args:
+  #
+  #   which_sig_list: a list of `whichSignature` output for each sample
```

Comment:
Personal preference here, but I kind of like named lists, rather than separate list items and an id vector.  For me, it does a better job of keeping things in sync. It also prevents possible errors if ids are repeated when they shouldn't be (which in this case would result in overwriting files...)

Reply:
Ah good point. Will change this.